### ISSUE TITLE
gh-pages support

### ISSUE THREAD

**Issue by stuartherbert** [datasift/gitflow#47]
Although gh-pages branches aren't really part of the original Gitflow model, they are a special type of branch that we end up dealing with when working with GitHub.

Hubflow needs to support the gh-pages branch:
- Support creating it on request
- Treat it as another 'master' branch
- Support a 'develop-doc' branch for pending doc changes
- Support creating 'feature-doc' branches
- Support creating corresponding 'doc' branches for features
- Support creating corresponding 'doc' branches for releases
- Support creating corresponding 'doc' branches for hotfixes

Main challenge will be to get the right balance between ease of use and how well it works in a multi-team environment.

**Comment by webmaven** [comment]
Any progress on this?

**Comment by stuartherbert** [comment]
No. I'm not likely to work on this any time soon, unless there's a lot of people out there who really need this functionality.

**Comment by abhisekp** [comment]
@stuartherbert I need this functionality. :smiling_imp:

**Comment by webmaven** [comment]
I could use this functionality as well.

**Comment by stuartherbert** [comment]
I'll see what I can do to get this into v1.7.

**Comment by webmaven** [comment]
Cool, thanks! :+1:
